src/test/java/xbot/edubot/rotation/GoToOrientationTest.java

```diff
@@ -11,7 +11,7 @@ public class GoToOrientationTest extends BaseOrientationEngineTest implements Se
 
     @Test
     public void testGoToOrientation0to150() {
-        DriveToOrientationCommand command = new DriveToOrientationCommand(this.drive);
+        DriveToOrientationCommand command = this.getInjectorComponent().driveToOrientationCommand();
```

Comment:
❔ out of curiosity, why move away from building the commands manually here?

Reply:
Students were getting super confused in the last exeecises when changing constructors.